config/routes.rb

```diff
@@ -1,6 +1,27 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_for :users
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+   # Commented to prevent defaulting to sign in view to check if signed in or not
+    devise_scope :user do
+      root 'devise/sessions#new'
+    end
+
+  resources :recipes
+  
+  resources :recipe_foods
+
+  resources :public_recipes
```

Comment:
- [ ] It will be great to ensure that you don't generate routes that are not used in the application. In your public recipe, I can see you have generated all the routes but you have only used the index route. Kindly use `only` or `expect` methods to remove unwanted routes